Pocket Casts Watch App/FiltersListViewModel.swift

```diff
@@ -0,0 +1,34 @@
+import Combine
+import Foundation
+import PocketCastsDataModel
+
+class FiltersListViewModel: ObservableObject {
+    @Published var isLoading: Bool = false
```

Comment:
I had Xcode main thread checker warning about this not being updated on a main thread. I'm not sure if it depends because of the publisher receiving on the main loop.

Reply:
This one, I'm not sure why it complains, all the other view model we have in the WatchOS project do the same.